src/6-distributions/index.md

```diff
@@ -0,0 +1,64 @@
+---
+date: 2016-01-29
+id: 6-distributions
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "6-distributions", link: "./" }
+tags: dist, distributions
+title: IPFS distributions
+author: Richard Littauer
+collection: posts
+---
+
+[![](!img/screenshot.png)](http://dist.ipfs.io/)
+
+[dist.ipfs.io](http://dist.ipfs.io/) is the new distributions page for IPFS. This is the new one-stop-shop for finding and downloading all official binaries that IPFS produces.
+
+## Project details
+
+Every distribution has a section, which includes:
+
+- The distribution name and a short description;
+- The current version number and release date;
+- A download button that detects your platform and automatically suggests the appropriate distribution for you;
+- A grid with download links for all supported platforms (operating system and architectures);
+- A `Changelog`, a link to a summary of all version changes;
+- An `All Versions`, a link to view and download previous versions.
+
+The `All Versions` link on each distribution shows directory listings for all the available versions, and a `versions` file ([example](http://dist.ipfs.io/go-ipfs/versions)). This file can be used by tools, such as [ipfs-update](http://dist.ipfs.io/#ipfs-update), to find all the available versions and download the latest.
+
+The directory listing of each version ([example](http://dist.ipfs.io/go-ipfs/v0.3.11)) has all the platform archives (`.zip` or `.tar.gz`), a `README.md` and a `dist.json` which describe the release for humans and machines. It is meant to be easily consumed and used by tools.
+
+## The Site
+
+The site itself is, unsurprisingly but still awesomely, hosted on IPFS. All of the binaries are also hosted on IPFS. This means that you can view it through your local gateway by running an IPFS daemon and directing your browser to __include address here__. Accessing it locally means that downloading versions (__ Of the site? Or of the distros?__) will be faster in the future. You can also view it with browser extensions, __this way__.
```

Comment:
- I would leave out that local version, as the hash changes over time and we will be left with an outdated version soon.
- No idea how the browser extensions work, is this needed?